github/PullRequest.py

```diff
@@ -890,6 +890,21 @@ def remove_from_assignees(self, *assignees):
         )
         self._useAttributes(data)
 
+    def update_branch(self, expected_head_sha):
+        """
+        :calls `PUT /repos/:owner/:repo/pulls/:pull_number/update-branch <https://developer.github.com/v3/pulls>`_
+        :param expected_head_sha: string
+        :rtype: None
+        """
+        assert isinstance(expected_head_sha, str)
+        post_parameters = {"expected_head_sha": expected_head_sha}
+        self._requester.requestJsonAndCheck(
+            "PUT",
+            self.url + "/update-branch",
+            input=post_parameters,
+            headers={'Accept': Consts.updateBranchPreview}
+        )
+
```

Comment:
Can you please follow the pattern in Repository and assert the status returned is 202?

Reply:
thx for that didn't know about it - done